app/services/iteration_service.rb

```diff
@@ -0,0 +1,59 @@
+class IterationService
+  DAYS_IN_WEEK = (1.week / 1.day)
```

Comment:
this is just so I don't have a magic number "7" below although it's pretty obvious that 7 means size of days in a week